util/util.go

```diff
@@ -16,3 +19,15 @@ func CheckErr(err error, msg string, exit int) {
 		}
 	}
 }
+
+// MonitoringStatus generates status from backup runs, which the prometheus
+// node exporter can pick up
+func MonitoringStatus(volume string) {
+	ts := time.Now().Unix()
+	stamp := fmt.Sprint(ts)
+	labels := fmt.Sprintf("volume=\"%s\",what=\"lastruntimestamp\"", volume)
+	metric := fmt.Sprintf("conplicity{%s} %s\n", labels, stamp)
```

Comment:
How about monitoring status? Maybe we could get it from the duplicity return code?